### ISSUE TITLE
Add 'concepts' section in the documentation.

### ISSUE THREAD

**Issue by chicken-biryani** [conda-forge/conda-forge.github.io#1665]
### Where should the content be added?

Not sure where. Maybe after the introduction.

### What should be added?

A newbie-- like me-- may not be aware of the terms 'recipe', 'feedstock', etc.

A section, namely 'concepts', that talks about these terms in a simple way without jargon may help newcomers understand things in a better way.

### Additional information

I am not aware of the target audience for the documentation. So this may be an irrelevant or a helpful issue.

**Comment by beckermr** [comment]
Go for it!

**Comment by BastianZim** [comment]
It definitely will but then again we can be sure that everything's included that a beginner might need to know. 

If you have any questions, just ping me.

**Comment by jaimergp** [comment]
Maybe this [glossary](https://jaimergp.github.io/scientific-packaging-glossary/) I started last summer can be useful!

**Comment by chicken-biryani** [comment]
Thank you @jaimergp.

Although, are there any sources that can help me get started with the basics. I.e well.. recipes.

(I know this may feel like spoon feeding, but I'll not need the spoon feeding soon- hopefully.)

**Comment by BastianZim** [comment]
I would start by writing out all concepts that you don't know and then we can add info there.  

Otherwise [this](https://docs.conda.io/projects/conda-build/en/latest/) may be a start but not everything applies to conda-forge.

**Comment by jaimergp** [comment]
Some links:

* Each feedstock's README contains this [blurb of text](https://github.com/conda-forge/plantuml-feedstock#about-conda-forge).
* Pangeo Forge is like conda-forge but for data. The [parallels](https://github.com/pangeo-forge/roadmap#vision) might be useful as a different way of looking at conda-forge.
* The Bioconda [FAQ](https://bioconda.github.io/contributor/faqs.html) is, again, not exactly applicable to conda-forge, but might provide yet another angle at the same problem.

**Comment by chicken-biryani** [comment]
Hello,

Although I haven't usually seen it in docs. Do you think- as a follow up- it's a good idea to add an overview/detailed flowchart of the whole process, end to end? This may help a person get a clearer picture.

**Comment by BastianZim** [comment]
I think that would be a great idea. Maybe even make it interactive and depending on the language. 

For example, if python is selected, recommend Grayskull etc.

**Comment by kathatherine** [comment]
I agree! Whatever might help a new user, either someone completely new to python and conda or someone just new to conda-forge to understand what they need to do to accomplish their goals (getting conda-forge packages, contributing to conda-forge, etc.) would be fantasic.

**Comment by chicken-biryani** [comment]
Can we add analogies while trying to explain a term? Eli5 (Explain like I am 5) analogies?

For example,

For packages... MAYBE how you get a package of a chair from IKEA... With instructions on how to build the chair and the parts of the furniture... Similarly for packages- they come with the source code and instructions.

If people relate to something in the non virtual world they understand things better.

I didn't see it in conda or conda-build's documentation... But do people? Or maybe that get's too informal and blogs may be a better place?
